process/block/shardblock.go

```diff
@@ -399,6 +411,21 @@ func (sp *shardProcessor) checkEpochCorrectness(
 	return nil
 }
 
+func (sp *shardProcessor) getOldEpochStartMetaHdr(header *block.Header) (data.HeaderHandler, error) {
+	metaBlock, err := sp.dataPool.Headers().GetHeaderByHash(header.EpochStartMetaHash)
```

Comment:
I think this line could be replaced with:
	metaBlock, err := process.GetMetaHeader(header.EpochStartMetaHash, sp.dataPool.Headers(), sp.marshalizer, sp.store)

In this way also header.EpochStartMetaHash would be searched first in pool and then if it is not found there in cacher->storage

Reply:
no need for that, as in case of storage it should be saved with the epoch trigger key.

Reply:
ok